src/MIISHandler/MIISHandler.csproj

```diff
@@ -13,7 +13,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>MIISHandler</RootNamespace>
     <AssemblyName>MIISHandler</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
I believe it does not have to be 4.6.1. I'm running it on 4.0 and it seems fine. However, the Markdig.SyntaxHighlight author seems to demand a higher version, which forced this.

Reply:
I'd rather prefer not to upgrade to .NET 4.x except if it's really mandatory. .NET 3.x is available even in older versions of Windows Server and tehrefore will make MIIS work out of the box in almost all installations. Sorry. You should add Syntax Highlighting to your template as explained in this issue: https://github.com/jmalarcon/MIIS/issues/8

That's the best way to go since it doesn't implies to change the product and you can add it or remove it in any templates that you need. I'll try to add this feature to the included templates in a next release, but it's pretty straight-forward.

Thanks.